commons/zenoh-protocol/src/proto/msg.rs

```diff
@@ -1187,7 +1175,7 @@ impl ZenohMessage {
         key: WireExpr<'static>,
         value_selector: String,
         qid: ZInt,
-        target: Option<QueryTAK>,
+        target: Option<QueryTarget>,
```

Comment:
Shouldn't QueryTarget be mandatory at this point? Seems odd to me that the protocol would allow not specifying any.

Reply:
Yes, but I wouldn't change it now. In the forthcoming version of the protocol default values will be treated as such and not as options.